WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -619,7 +619,11 @@ private void viewPosts() {
 
     private void viewPages() {
         requestNextStepOfActiveQuickStartTask();
-        ActivityLauncher.viewCurrentBlogPages(requireActivity(), getSelectedSite());
+        SiteModel selectedSite = getSelectedSite();
+        if (selectedSite != null) {
+            ToastUtils.showToast(getActivity(), R.string.site_cannot_be_loaded);
```

Comment:
Shouldn't this line be in an `else` clause? Wdyt?

Reply:
Good catch, I've fixed it in https://github.com/wordpress-mobile/WordPress-Android/pull/10826/commits/60bcddc5f23f2dbf7d73d95142fbf13964814ea6